backend/app/api/routers/graph.py

```
@@ -0,0 +1,525 @@
+from app.database_neo4j import Neo4jManager, get_neo4j, Neo4jUnavailable
+
+
+LOGGER = logging.getLogger(__name__)
+router = APIRouter(prefix="/graph", tags=["graph"])
+
+
+# Valid entity types (spaCy NER types)
+VALID_ENTITY_TYPES = {
+    "PERSON", "ORG", "LOC", "DATE", "PRODUCT", "GPE", "EVENT",
+    "WORK_OF_ART", "LAW", "LANGUAGE", "NORP", "PERCENT", "MONEY",
+    "QUANTITY", "ORDINAL", "CARDINAL", "TIME", "FAC", "ORG",
```

Ruff (0.14.10)

Sets should not contain duplicate item `"ORG"`

Remove duplicate item

(B033)